sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ServiceBusMultiSessionProcessorSample.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.messaging.servicebus.models.ReceiveMode;
+import com.azure.messaging.servicebus.models.ServiceBusErrorContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class ServiceBusMultiSessionProcessorSample {
+
+    public static void main(String[] args) {
+        final MyMessageProcessor myMessageProcessor = new MyMessageProcessor();
+        ServiceBusMultiSessionProcessorClient multiSessionProcessorClient = new ServiceBusClientBuilder()
+            .connectionString("connectionString")
+            .multiSessionProcessor()
```

Comment:
Why do you need to specify multisession here, and then again when you build() at the end?

Reply:
I certainly have not put enough time on naming this class.
This duplication should be avoided, I should change `multiSessionProcessor()` to `buildProcessor()`